### DISCUSSION TITLE
2022-11-05-pake

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#123] [General]
# 用 Pake 打包了一个极简 Twitter - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Life一个很简单的用 Rust 打包网页生成很小的 Mac App 工具，底层使用 Tauri，当前支持微信读书、Twitter、语雀、RunCode、Witeboard、Flomo、Vercel 等。

https://tw93.fun/2022-11-05/pake.html

**Comment by jincaiw** [comment]
你的这个博客，看起来特别简洁，能分享一下用什么代码建设的吗？

**Reply by tw93** [reply]
点击地步 有一个github地址
7年前刚学完前端弄的 说实话技术一般般

**Reply by whsheng** [reply]
v5，了解到Pake，还是在twitter上面通过一个黄推看到的！

**Reply by ZhangShangyu** [reply]
博客的字体好有感觉，请问是啥字体呢

**Comment by xiyuvi** [comment]
我挂了梯子还是一直在downloading rust，请问有什么方法可以加快这个过程嘛

**Reply by tw93** [reply]
直接绑定国内rust镜像，去搜索下就知道了

**Comment by weiqifa0** [comment]
🆒

**Comment by venomwise** [comment]
唠嗑😭😭😭
